### PR TITLE
view-average-grade | James Chiu

### PR DESCRIPTION
**Pull request by jachiu555**
SGT view-average-grade SS

### CODE REVIEW

File: components/app.js [hunk 1]
```diff
@@ -12,6 +13,17 @@ class App {
...
   handleGetGradesSuccess(grades) {
     this.gradeTable.updateGrades(grades);
+
+    let totalGrade = 0;
+    let averageGrade = 0;
+
+    for (let i = 0; i < grades.length; i++) {
+      totalGrade += grades[i].grade;
+
+      averageGrade = totalGrade / grades.length;
```
Comment (scbowler) [inline components/app.js]: Although this probably ultimately works, this calculation doesn't need to be done on every iteration of the `for` loop. The calculation should only be done once after the `for` loop has completed.

File: index.html [hunk 2]
```diff
@@ -27,6 +30,7 @@ <h1 class="text-left">Student Grade Table</h1>
 
   </body>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.js"></script>
+  <script src="components//page-header.js"></script>
```
Comment (scbowler) [inline index.html]: You have an extra forward slash `/` in-between `components` and `page-header.js`

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good but a couple small things need fixed, see below comments for details.

**Review by WhiskeyJack96 — APPROVED** [review]
Great!

**Pull request merged**